.github/workflows/README.md

```diff
@@ -0,0 +1,83 @@
+# DevOps Apprenticeship: Project Exercise
+
+## System Requirements
+
+The project uses poetry for Python to create an isolated environment and manage package dependencies. To prepare your system, ensure you have an official distribution of Python version 3.7+ and install Poetry using one of the following commands (as instructed by the [poetry documentation](https://python-poetry.org/docs/#system-requirements)):
+
+### Poetry installation (Bash)
+
+```bash
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
+```
+
+### Poetry installation (PowerShell)
+
+```powershell
+(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py -UseBasicParsing).Content | python -
+```
+
+## Dependencies
+
+The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from your preferred shell:
+
+```bash
+$ poetry install
+```
+
+You'll also need to clone a new `.env` file from the `.env.template` to store local configuration options. This is a one-time operation on first setup:
+
+```bash
+$ cp .env.template .env  # (first time only)
+```
+
+The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+
+
+## new Environment variables
+
+SECRET_KEY=secret-key
+TRELLO_BOARDID=trello-id
+TRELLO_KEY=trello-key
+TRELLO_TOKEN=trello-token
+TRELLO_LISTID1=trello-listid1
+TRELLO_LISTID2=trello-listid2
```

Comment:
Don't forget to clear out all the old references to Trello that are no longer relevant 🙂

Reply:
done